sunpy/map/map_factory.py

```diff
@@ -168,10 +168,13 @@ def _read_file(self, fname, **kwargs):
         for pair in pairs:
             filedata, filemeta = pair
             assert isinstance(filemeta, FileHeader)
+            meta = MetaDict(filemeta)
+            # If the header is an extension, we want to do ignore, maybe?
+            if "xtension" in meta:
+                continue
```

Comment:
Terrible idea but I got nothing better right now.

Reply:
This is an improvement but probably still isn't great.
```suggestion
            if meta.get("XTENSION") == "TABLE":
                continue
```